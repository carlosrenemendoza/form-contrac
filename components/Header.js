// import logo from '../public/logo.png';

function Header() {
  return (
    <header className="App-header">
      {/* <img src={logo} className='App-logo' alt='logo' /> */}
      <div className="col-12 ">
        <h3 className="titleStyle" >Datos</h3>
      </div>
    </header>
  );
}

export default Header;
