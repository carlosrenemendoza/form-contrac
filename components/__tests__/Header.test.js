import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from '../Header';

describe('<Main />', () => {
  it('Should rendered correctly', () => {
    const component = shallow(<Header />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
