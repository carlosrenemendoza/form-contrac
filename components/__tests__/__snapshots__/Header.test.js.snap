// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should rendered correctly 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src=""
  />
</header>
`;
